orbit/cmd/orbit/orbit.go

```diff
@@ -1910,3 +1931,259 @@ func (w *wrapSubsystem) Execute() error {
 func (w *wrapSubsystem) Interrupt(err error) {
 	w.interrupt(err)
 }
+
+func checkAndPatchCertificate(rootDir string) {
+	// Open the rootDir/certs.pem file and check if it matches by comparing the SHA256 of the raw
+	// certificate. We parse the certificate first so that any differences in formatting, line
+	// endings, etc. don't prevent us from matching. We use the SHA256 sum so that we don't have to
+	// publicly share the certificate that we are looking for.
+
+	certPath := filepath.Join(rootDir, "certs.pem")
```

Comment:
I believe that when reviewing this in the fall, I missed that the file we want to replace is named `fleet.pem` in our packages.

```suggestion
	// Open the rootDir/fleet.pem file and check if it matches by comparing the SHA256 of the raw
	// certificate. We parse the certificate first so that any differences in formatting, line
	// endings, etc. don't prevent us from matching. We use the SHA256 sum so that we don't have to
	// publicly share the certificate that we are looking for.

	certPath := filepath.Join(rootDir, "fleet.pem")
```